examples/oemof_0.2/excel_reader/dispatch.py

```diff
@@ -16,14 +16,17 @@
 
 Installation requirements
 -------------------------
-This example requires the latest version of oemof, matplotlib and xlrd. Install
-by:
+This example requires the latest version of oemof and others.
+Install by:
 
-    pip install 'oemof<0.3,>=0.2'
-    pip install xlrd
-    pip install matplotlib
+    pip3 install 'oemof<0.3,>=0.2'
```

Comment:
We strongly recommend to use oemof within a virtual environment. You will not need a "3" there but it does not harm.